src/panels/ai/SelectionPanelSection.tsx

```diff
@@ -19,18 +31,14 @@ export function SelectionPanelSection() {
       {selection && (
         <div className="mb-6 flex flex-col gap-4">
           <div className="border border-slate-100 bg-slate-50 p-4">{debouncedSelection}</div>
-          <button
-            className="btn-primary"
-            disabled={rewriteCallState.state === 'loading'}
-            onClick={() => callRewrite(debouncedSelection)}
-          >
+          <button className="btn-primary" disabled={isFetching} onClick={fetchRewrite}>
             REWRITE
           </button>
-          {rewriteCallState.state === 'error' && <span className="bg-red-50">{String(rewriteCallState.error)}</span>}
-          {rewriteCallState.state === 'loading' && <span>Processing...</span>}
-          {rewriteCallState.state === 'ok' && (
+          {error ? <span className="bg-red-50">{String(error)}</span> : null}
```

Comment:
Why not write `!!error && <span className="bg-red-50">{String(error)}</span>` here?

Reply:
Updated.